src/read.c

```diff
@@ -128,10 +130,23 @@ static UInt GlobalComesFromEnclosingForLoop(ReaderState * rs, UInt var)
     return 0;
 }
 
+static void Match_(ReaderState * rs,
```

Comment:
If somebody has a better idea for a good name for this static wrapper function, please let me know

Reply:
TODO: should add a documentation comment for this function